### PR TITLE
added verification of the existing setup of google cloud project

### PR DESCRIPTION
**Pull request by SantoshDhirwani**
## Description
Added flag in the main script to get the user's input: 1 or 0
(0: use existing setup, 1: clean existing setup and a create a new one)

When the user selects 1: Clean the existing setup and create a new one

When the user selects 0: Check whether the number of nodes mentioned by the user in the config file is equal to the number of running VMs (in google cloud project) or not.

## Related Issue
#91 

## Motivation and Context
It has been added to verify the existing configuration first and then use it.
Also, it is not a good idea to create a new setup without permission.

## How Has This Been Tested?
By executing the following in the bin folder:
python3 main.py 2 0 (when the number of nodes in config file = number of running VMs on GCP)
python3 main.py 2 0 (when the number of nodes in config file != number of running VMs on GCP)
python3 main.py 2 1

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist contributor:
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: bin/sut/deploy-sut.sh [hunk 1]
```diff
@@ -4,11 +4,12 @@
 
 set -e
 
+#################################################################################
+
 NUMBER_NODES=${1}
 BLOCK_INTERVAL=${2}
 BLOCK_SIZE=${3}
 NEW_SETUP=${4}
-GCLOUD_OUTPUT=${5}
```
Comment (fayanpardo) [inline bin/sut/deploy-sut.sh]: Why has this been deleted?

**Review by fayanpardo — CHANGES_REQUESTED** [review]
I do not understand the execution. 
If I run main.py 2 0 without a SUT built I get the following output.
image

Successfully built? benchmark done and found minimum interval? there are no VMs running in GCP. An it just keeps running and failing but for the user seems that is working. Either the messages are not updated and clear or something else is failing.

If I run instead main.py 2 1. It builds and deletes and builds and deletes VMs constantly for every benchmark.
image

Either is not working or either is wasting a lot of time for something that was already working.

**Comment by SantoshDhirwani** [comment]
@boxrhcp I think the previous approach was a bit complex. I added a simpler approach.

**Comment by ulalaa14** [comment]
i run a test with `python3 main.py 2 2` but it still running the command. even though it only handle parameter input 0 or 1 .
image
image

**Comment by SantoshDhirwani** [comment]
@ulalaa14  Nice catch. Added it.

**Review by fayanpardo — DISMISSED** [review]
works! gj!

**Comment by SantoshDhirwani** [comment]
@boxrhcp  There were some conflicts with the master in the ReadMe. I resolved them
 Could you please approve it again? Thanks.

**Review by fayanpardo — APPROVED** [review]

**Pull request merged**